app/models/song.rb

```diff
@@ -0,0 +1,4 @@
+class Song < ApplicationRecord
+  belongs_to :playlist
+  belongs_to :user
```

Comment:
Did you intend to have both `Playlist` and `Song` belong to `User` (IE each could belong to a different user?), or should `Song` belong to `User` THROUGH `Playlist`?